pom.xml

```diff
@@ -71,18 +73,18 @@
 		<stringtemplate.version>4.0.2</stringtemplate.version>
 		<open-ai-client.version>0.16.0</open-ai-client.version>
 		<azure-open-ai-client.version>1.0.0-beta.3</azure-open-ai-client.version>
```

Comment:
I was told by some folks at AWS that there is a newer release than the 20.157; please make sure to use a more recent, cleaned-up, and better-documented version of the bedrock SDK: https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/bedrock/package-summary.html.

Reply:
This is good, thank you!